index.html

```diff
@@ -30,16 +44,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 1</div>
```

Comment:
Good job targeting specific divs by assigning them their own class. As you progress you'll learn more Selectors and adding multiple classes to each element.

Reply:
so instead of renaming the div class I should have added colors as part of the class, like this?

<div class="box">Box8</div>
      <div class="gold">Box8</div> (just using gold as an example).